### DISCUSSION TITLE
First East Asian site? (gratitude actually)

### DISCUSSION THREAD

**Discussion by GJRobert** [oleeskild/obsidian-digital-garden#269] [Show and tell]
Not much of showing off, but a million thanks to Digital Garden for Obsidian!

My first Digital Garden site (more planned): [資訊與網路的恩典 (literally "Amazing Grace of IT and Internet")](https://imazingrace.vercel.app/)
I'm from Taiwan and I write mostly in some Taiwanese languages and Chinese, so you can see some beautiful East Asian characters on my pages.
![imazingrace vercel app](https://user-images.githubusercontent.com/655584/231825223-7df57185-7fb0-4a74-b85d-c8ba5bdbfa72.png)
(theme: Cyber Glow)

Not much content by now because my main digital garden is still a local DokuWiki site, but I have to admit that it's happy to use Obsidian + Digital Garden.

I am even using a crude way to switch among sets of obsidian-digital-garden plugin settings, so to publish different sets of pages to different Digital Garden sites just within the same Obsidian vault. (will share my method for more feedback soon)

**Comment by shinanqu** [comment]
Very nice. Good luck with your Digital Garden!
